assessment.md

```diff
@@ -82,39 +82,75 @@ TODO: add description for each assessment level for this skill.
 
 * Using the GitHub Flow model
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Never has used the GitHub flow model before.
+  * :worried: = Uses it occasionally, but at inappropriate times.
+  * :neutral_face: = Student uses it often, sometimes appropriately, but sometimes inappropriately as well.
```

Comment:
I think it would be great to elaborate on what is the "inappropriate" use of the GitHub flow model. It could be using unclear or inappropriate commit messages, merging PRs without getting approval from TLs and instructor, deleting other groups' branches... etc. Each one of these uses should be somewhat addressed in the assessment criteria,  not just for `Using the GitHub Flow model` but for others as well

Reply:
Looking back at my comment after reading through your addition. It looks like you've addressed some of it but there are still few things missing. I think that giving examples of what is appropriate and inappropriate uses is still needed. Feel free to ask questions if you need me to clarify my change requests. Thanks